docs/cli/discount-code-generator.md

```diff
@@ -0,0 +1,153 @@
+# Discount Code Generator
+
+Generate unique discount codes to be imported to the [commercetools platform](https://dev.commercetools.com/). See usage below
+
+## Usage
+`npm install @commercetools/discount-code-generator --global`
+
+### CLI
+```
+Usage: discountCodeGenerator [options]
+
+Options:
+  --help, -h                 Show help text.                                                     [boolean]
+  --version, -v              Show version number.                                                [boolean]
```

Comment:
>does -v accepts Boolean? what happens if false is passed in?

Well it does, and if false is passed in, it ignores the flag. Surprisingly, this is the default behaviour of yargs (even our other cli tools using yargs have this behaviour) 🤔